### ISSUE TITLE
Variable Attributes: only one available attribute captured

### ISSUE THREAD

**Issue by iv-mexx** [krzysztofzablocki/Sourcery#540]
> What version of Swift are you using (swift --version)?

Apple Swift version 4.0.3 (swiftlang-900.0.74.1 clang-900.0.39.2)
Target: x86_64-apple-macosx10.9

> What did you do?

I'm parsing variables of a protocol to generate an implementation of the protocol. It is necessary to specify and keep `@available` attributes. 

This is a minimal Version of the Template I'm trying to build:

```
{% for protocol in types.protocols %}
extension MyExtension {
{% for variable in protocol.variables %}
  {{ variable.attributes.available}}
  {{ variable.name }}
{% endfor %}
}
{% endfor %}
```

This is a minimal Code Sample that is going to be parsed:

```swift
protocol MyProtocol {
  @available(tvOS 10, *)
  @available(iOS 10, *)
  var badgeColor: UIColor? { get }
}
```

> What did you expect to see?

I'd expect both availability annotations to be preserved in the generated code

```swift
extension MyExtension {
  @available(tvOS 10, *)
  @available(iOS 10, *)
  badgeColor
}
```

> What did you see instead?

Only the last availability annotation was preserved

```swift
extension MyExtension {
  @available(iOS 10, *)
  badgeColor
}
```

**Comment by ilyapuchka** [comment]
Hi @iv-mexx. Multiple attributes of the same type are not supported, but in this particular case you can mention several platforms in one attribute, i.e. `@available(iOS 10.0, macOS 10.12, *)`. See FileParser+AttributesSpec.swift:60 for example.

**Comment by iv-mexx** [comment]
Alright, thanks!
